core/remoteviewserver.cpp

```diff
@@ -91,7 +93,6 @@ bool RemoteViewServer::isActive() const
 
 void RemoteViewServer::sendFrame(const RemoteViewFrame &frame)
 {
-    m_clientReady = false;
```

Comment:
Hm I don't think so.
If we reach requestUpdateTimeout a QuickOverlay grab request is triggered.
Depending at which time the request is done, we can already be in locked state in QuickOverlay::windowAfterRendering which will emit a sceneChanged just before we can change the grabbing mode. This will then trigger a new update request while we did not even yet changed the QuickOverlay grabbing mode. As a result the next request timeout will try to request a frame grab but **could** do nothing because we **could** be in the process of capturing a frame already (ie, grabbing mode is true already).
Schematized by: "Request -> Request -> Grab" **instead of** "Request -> Grab -> Request -> Grab"
So for me if the client requested a frame but did not yet received it, I think a request update can't occurs server side as normally only a clientViewUpdated call could trigger a possible frame grab (assaying - client is ready).
As a consequence, requesting a frame update should mark client as non ready.
If my reasoning sound wrong to you, then probably we want to fix the double update request calls (ie, by introducing a quickoverlay grabbing state test in RemoteViewServer::checkRequestUpdate() so the new request will only occurs after the current grab finished).